cvat/apps/engine/media_extractors.py

```diff
@@ -72,6 +72,48 @@ def save_image(self, k, dest_path):
         image.close()
         return width, height
 
+class PDFExtractor(MediaExtractor):
```

Comment:
I would look at Archive implementation and inherit the class from ImageListExtractor. Let's implement here _extract method ... What do you think?

Reply:
I was using `VideoExtractor` as a basis here because in my case, PDF's could have multiple pages. Is there a better way to do this where it could have multiple pages?